src/views/UserSettings.js

```diff
@@ -0,0 +1,438 @@
+        Manage Organizations &nbsp;
+        <Tooltip
+          title={
+            <p style={{ textAlign: 'center' }}>
+              View your existing organization memberships and leave if you
+              choose
+            </p>
+          }
+          placement="right"
+        >
+          <QuestionCircleTwoTone style={{ fontSize: '0.8em' }} />{' '}
+        </Tooltip>
+      </h1>
+      <Search
+        style={{ width: '50%', marginBottom: '20px' }}
+        placeholder="User Search"
+        enterButton
+        onSearch={console.log}
+      />
+      <Table
+        columns={columns}
+        dataSource={organizations}
+        onChange={onChange}
+        pagination={{ pageSize: 10 }}
+        scroll={{ y: 240 }}
+      />
+    </Card>
+  );
+}
+
+function Sidebar() {
+  return (
+    <Affix offsetTop={60}>
+      <Sider width={250}>
+        <Menu
+          mode="inline"
+          theme="light"
+          defaultOpenKeys={['users', 'resources']}
+          style={{ height: '100%', borderRight: 0 }}
+        >
+          <Menu.Item
+            key="dashboard"
+            icon={<AreaChartOutlined />}
+            style={{ marginTop: '30px' }}
+            onClick={() => {
+              window.scrollTo({
+                top: 0,
+                behavior: 'smooth',
+              });
+            }}
+          >
+            User Overview
+          </Menu.Item>
+          <Menu.Item
+            key="organizations"
+            icon={<TeamOutlined />}
+            onClick={() => {
+              window.scrollTo({
+                top: 250,
+                behavior: 'smooth',
+              });
+            }}
+          >
+            Organizations
+          </Menu.Item>
+          <Menu.Item
+            key="pending"
+            icon={<FileProtectOutlined />}
+            onClick={() => {
+              window.scrollTo({
+                top: 600,
+                behavior: 'smooth',
+              });
+            }}
+          >
+            Uploaded Resources
+          </Menu.Item>
+        </Menu>
+      </Sider>
+    </Affix>
+  );
+}
+
+function UserSettings() {
+  let [user, setUsers] = useState([]);
+  useEffect(() => {
+    API.get('/api/users/').then(setUsers);
+  }, []);
+  return (
+    <Layout style={{ backgroundColor: '#fff' }}>
+      <Row justify="start" align="middle">
+        <Col span={3}>
+          <a href="/" style={{ margin: '15px' }}>
+            <img alt="logo" src="/logo.png" width={'160px'} />
+          </a>
+        </Col>
+        <Col span={17}>
+          <Breadcrumb style={{ marginLeft: '20px' }}>
+            <Breadcrumb.Item>
+              <a href="/">Home</a>
+            </Breadcrumb.Item>
+            <Breadcrumb.Item>
+              <a href="/">User Name</a>
+            </Breadcrumb.Item>
+            <Breadcrumb.Item>Settings</Breadcrumb.Item>
+          </Breadcrumb>
+        </Col>
+        <Col span={4}>
+          <LoginButton />
+        </Col>
+      </Row>
```

Comment:
The same comment for this header. I can see us wanting to reuse this a lot, also in the future (or now if you think it'll be easy), the component could also read from something like "window.location" and create the breadcrumbs automatically.